pkg/proxy/ipvs/README.md

```diff
@@ -200,15 +200,50 @@ DNAT       tcp  --  0.0.0.0/0            0.0.0.0/0            /* default/nginx-s
 Currently, local-up scripts, GCE scripts and kubeadm support switching IPVS proxy mode via exporting environment variables or specifying flags.  
 
 ### Prerequisite
-Ensure the following kernel modules required by IPVS-based kube-proxy have been compiled into the node kernel (use `lsmod` to check):
-
+Ensure IPVS required kernel modules
 ```shell
 ip_vs
 ip_vs_rr
 ip_vs_wrr
 ip_vs_sh
 nf_conntrack_ipv4
 ```
+1. have been compiled into the node kernel. Use
+
+`grep -e ipvs -e nf_conntrack_ipv4 /lib/modules/$(uname -r)/modules.builtin`
+
+and get results like the followings if compiled into kernel.
+```
+kernel/net/ipv4/netfilter/nf_conntrack_ipv4.ko
+kernel/net/netfilter/ipvs/ip_vs.ko
+kernel/net/netfilter/ipvs/ip_vs_rr.ko
+kernel/net/netfilter/ipvs/ip_vs_wrr.ko
+kernel/net/netfilter/ipvs/ip_vs_lc.ko
+kernel/net/netfilter/ipvs/ip_vs_wlc.ko
+kernel/net/netfilter/ipvs/ip_vs_fo.ko
+kernel/net/netfilter/ipvs/ip_vs_ovf.ko
+kernel/net/netfilter/ipvs/ip_vs_lblc.ko
+kernel/net/netfilter/ipvs/ip_vs_lblcr.ko
+kernel/net/netfilter/ipvs/ip_vs_dh.ko
+kernel/net/netfilter/ipvs/ip_vs_sh.ko
+kernel/net/netfilter/ipvs/ip_vs_sed.ko
+kernel/net/netfilter/ipvs/ip_vs_nq.ko
+kernel/net/netfilter/ipvs/ip_vs_ftp.ko
+```
+
+OR
+
+2. have been loaded.
+ ```shell
+# to check loaded modules, use
+lsmod | grep -e ipvs -e nf_conntrack_ipv4
+# or
+cut -f1 -d " "  /proc/modules | grep -e ip_vs -e nf_conntrack_ipv4
+
+# load module <module_name>
+modprobe -- <module_name>
```

Comment:
kube-proxy will attempt to run `modprobe` before looking in `/proc/modules`, so pre-loading the modules is a strict requirement. https://github.com/kubernetes/kubernetes/blob/b3837d004adab8e1f5f0eae7fdd2ddcd614258a0/pkg/proxy/ipvs/proxier.go#L464-L471

Reply:
@detiber Got it. Thanks :)